### DISCUSSION TITLE
Hiding or deleting media from a playlist

### DISCUSSION THREAD

**Discussion by dagandrew** [sircharlo/meeting-media-manager#2767] [Ideas] [answered]
you know that particular video or picture won't be shown, can you just hide from the playlist? (you don't have to put the trash icon next to every video, but maybe right click and hide or detete?

**Comment by sircharlo** [comment]
Great idea! I'll work on this.

**Comment by sircharlo** [comment, marked as answer]
Implemented in 91fd85232f1f1c57489c5d1879dd716c039abb9d. Try right-clicking a media item in v24.10.10!

**Comment by dagandrew** [comment]
Works like a charm. Thank you!
